src/services/users/index.js

```diff
@@ -7,5 +7,5 @@ import findOne from './findOneUser';
  * Remember to remove the default keyword since you are exporting two or more variables
  */
 export default {
-  findOne,
+  findOne
 };
```

Comment:
This file does not follow team convention, we now have it as `./src/modules/users/users.services.js` try remove it